Adjust/ADJUserDefaults.m

```diff
@@ -164,7 +135,6 @@ + (void)clearAdjustStuff {
     [[NSUserDefaults standardUserDefaults] removeObjectForKey:PREFS_KEY_DEEPLINK_URL];
     [[NSUserDefaults standardUserDefaults] removeObjectForKey:PREFS_KEY_DEEPLINK_CLICK_TIME];
     [[NSUserDefaults standardUserDefaults] removeObjectForKey:PREFS_KEY_DISABLE_THIRD_PARTY_SHARING];
-    [[NSUserDefaults standardUserDefaults] removeObjectForKey:PREFS_KEY_IAD_ERRORS];
```

Comment:
Shouldn't the data generated by previous versions be cleared?

Reply:
In theory it could have be left in place as well, but since this part of the code is not actually running in production (but only for internal end to end testing logic) and in current setup, it doesn't make much difference for our purpose if this line is removed or left in `clearAdjustStuff` method.